addons/binding/org.openhab.binding.coolmasternet/README.md

```diff
@@ -0,0 +1,42 @@
+# CoolMasterNet binding
+
+The CoolMasterNet binding is used to control [CoolMasterNet HVAC bridge devices](https://coolautomation.com/products/coolmasternet/), using the "ASCII I/F" plaintext TCP control protocol.
+
+## Discovery
+
+The CoolMasterNet protocol does not support automatic discovery.
+
+## Thing Configuration
+
+* `controller` is a OpenHAB "bridge", and represents a single CoolMasterNet device. A single controller supports one or more HVAC units.
+
+* `hvac` is an HVAC device connected to a controller. Each `hvac` thing is identified by a CoolMasterNet UID (refer to CoolMasterNet controller documentation).
+
+Example demo.things configuration for two HVAC devices connected to a CoolMasterNet device found at IP 192.168.0.100:
+
+```
+Bridge coolmasternet:controller:main [ host="192.168.0.100" ] {
+  Thing hvac a [ uid="L1.100" ]
+  Thing hvac b [ uid="L1.101" ]
+}
+```
+
+## Channels
+
+* on
```

Comment:
Could you transform this to a small table that mentions the item type that you can use with these channels and a short description of each? Check how this is done in other bindings.